modules/olmv1-semver-support.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+// * operators/olm_v1/arch/olmv1-operator-controller.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="olmv1-semver-support_{context}"]
+= Support for semantic versioning
+
+Support for link:https://semver.org/[semantic versioning (semver)] is enabled in {olmv1} by default. Operator and extension authors can use the semver standard to define compatible updates.
+
+According to the semver standard, releases within a major version number, such as `>=1.0.0, <2.0.0`, must be compatible. As a result, Operator or extension authors do not need to define update edges explicitly. {olmv1-first} can use an Operator or extension's version number to determine if an update can be resolved successfully.
+
+Cluster administrators can easily define the update edges for Operators and extensions that follow the semver standard by using comparison strings to define to specify a desired version range.
```

Comment:
Same comment as above. Users aren't defining the upgrade edges, they're just specifying a range for versions that may or may not be valid to upgrade to based on the controller's resolution.